staging/src/k8s.io/client-go/tools/clientcmd/config.go

```diff
@@ -68,7 +69,10 @@ func (o *PathOptions) GetEnvVarFiles() []string {
 		return []string{}
 	}
 
-	return filepath.SplitList(envVarValue)
+	fileList := filepath.SplitList(envVarValue)
+	// prevent the same path load multiple times
+	ss := sets.NewString(fileList...)
+	return ss.List()
```

Comment:
we loose the order here. Does it matter?

Reply:
@sttts the order is important, I should not loose the order, I will fix it.

Reply:
What are the downsides of parsing the same file multiple times? Just thinking about complexity of keeping the order now or just to ignore the fact that we parse the same file multiple times....

Reply:
As the issue said, we can not switch context and some other operation, and we should check it to prevent unexpected error in the Subsequent code.